zend/value.cpp

```diff
@@ -401,7 +406,7 @@ Value& Value::operator=(struct _zval_struct* value)
         {
             // If reference count is greater than 1, we need to separate zval
             // This is the optimized version of SEPARATE_ZVAL_NOREF()
-            if (Z_COPYABLE_P(to))
+            if (Z_COPYABLE_P(to)))
```

Comment:
Without this ridiculous addition I would get a compiler error:
```
zend/value.cpp:410:13: error: expected ')' before '{' token
```
However I couldn't figure out how it got to be so wrong.

Reply:
It's probably a bug...

`#define Z_COPYABLE(zval)			((Z_TYPE(zval) == IS_ARRAY)`

[I've opened a bug on the PHP bug tracker.](https://bugs.php.net/bug.php?id=76820)

Reply:
My fix has been merged into the engine so I'm going to resolve conflicts and merge this in.